reframe/frontend/runreport.py

```diff
@@ -158,18 +159,24 @@ def junit_xml_report(json_report):
     '''Generate a JUnit report from a standard ReFrame JSON report.'''
 
     xml_testsuites = etree.Element('testsuites')
+    isotime = parser.parse(json_report['session_info']['time_start'][:-6])
```

Comment:
I don't think that this is needed. You should be able to do this with the standard library of Python.